core/src/banking_stage/transaction_scheduler/receive_and_buffer.rs

```diff
@@ -244,6 +244,10 @@ impl SanitizedTransactionReceiveAndBuffer {
                     .zip(fee_budget_limits_vec.drain(..))
                     .zip(check_results)
                     .filter(|(_, check_result)| check_result.is_ok())
+                    .filter(|((((_, tx), _), _), _)| {
+                        Consumer::check_fee_payer_unlocked(&working_bank, tx, &mut error_counts)
+                            .is_ok()
+                    })
```

Comment:
Looks like `check_fee_payer_unlocked()` is called at the end of filtering process, just to clarify that this PR does not result making additional calls of `check_fee_payer_unlocked` (to packets would be dropped before scheduler_controller calls it again).

Reply:
Also, do we still need to check fee payer in same frequency at scheduler_controller (and unprocessed_transaction_storage)?